src/figure.rs

```diff
@@ -230,10 +230,56 @@ impl Figure
 		self
 	}
 
-	/// Closes the gnuplot process.
-	///
-	/// This can be useful if you're your plot output is a file and you need to
-	/// that it was written.
+	/// Save the figure to a png file
+	/// # Arguments
+	/// * `file_path` - Path to the output file (png)
+	pub fn save_to_png(&mut self, file_path: &str)
+	{
+		self.set_terminal("pngcairo", file_path);
+		self.show().close();
+		self.set_terminal("wxt", &"");
```

Comment:
Could you restore the same terminal/output path that was set earlier? Those are stored in the self.terminal and sell.output_file.